crates/proto/src/quic/quic_client_stream.rs

```diff
@@ -343,11 +343,11 @@ impl<S: DnsUdpSocket + QuicLocalAddr> Debug for QuinnAsyncUdpSocketAdapter<S> {
     }
 }
 
-/// TODO: Naive implementation. Look forward to
+/// TODO: Naive implementation. Look forward to future improvements.
```

Comment:
I found this cliffhanger comment while I was here and made a best effort guess at what it was supposed to say :crystal_ball: